charts/opensearch/values.yaml

```diff
@@ -529,3 +529,8 @@ extraObjects: []
   #      selector:
   #        matchLabels:
   #          {{- include "opensearch.selectorLabels" . | nindent 6 }}
+
+serviceMonitor:
```

Comment:
LGTM, thanks @shubham-cmyk, please take care of the following.

1) Update the chart readme with some details, better to mention to install the prometheus CRD's as prerequisite. 
2) Add a comment in the `values.yaml` file above the `serviceMonitor` line.
3) Please Update the `CHANGELOG.md` file. (Sample PR https://github.com/opensearch-project/helm-charts/pull/569).
4) This is a very good change can we have this to the OpenSearch Dashboards also?

I will go ahead and merge this PR once the above points are addressed.

Thank you 
@TheAlgo @getsaurabh02 @peterzhuamazon

Reply:
Agreed with @prudhvigodithi , awaiting contributor to address above before giving my approval as well.

Thanks.

Reply:
Hi @prudhvigodithi and @peterzhuamazon 

Thank you so much for your feedback! 😊 I'm currently working on the requested changes.
Thanks again for your help and patience!

Reply:
We have pushed the required changes please continue to merge this :)